src/cpp/runner/OutputSave.cpp

```diff
@@ -17,20 +17,22 @@ std::string jacobian_file_name(const std::string& output_prefix, const std::stri
 void save_time_to_file(const std::string& filepath, const double objective_time, const double derivative_time)
 {
     std::ofstream out(filepath);
+
     out << std::scientific << objective_time << std::endl << derivative_time;
     out.close();
 }
 
 void save_value_to_file(const std::string& filepath, const double& value)
 {
-    std::ofstream out(filepath);
+    precise_ofstream<std::remove_reference_t<decltype(value)>> out(filepath);
+
     out << std::scientific << value;
```

Comment:
Do we need `out << std::scientific` here? It seems to be already done in the constructor of `precise_ofstream<T>`.